recipes/mprod-package/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "mprod-package" %}
+{% set version = "0.0.1a1" %}
+{% set sha256 = "7b2c969d98bb8e266f168a9ac5ceaba80127d87890162eb0f5f4f48c3f5eace7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mprod-package-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  skip: win 
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed . -vv
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+  host:
+    - pip
+    - python
```

Just use `host` here:
```suggestion
  host:
    - pip
    - python >=3.6
    - setuptools
```

---

Hey @dopplershift 

After committing this suggestion, I get both linter complaints and build failure